Swiftfin/Views/PagingLibraryView/PagingLibraryView.swift

```diff
@@ -251,27 +258,59 @@ struct PagingLibraryView<Element: Poster>: View {
         .proxy(collectionVGridProxy)
     }
 
-    // MARK: body
-
-    // TODO: becoming too large for typechecker during development, should break up somehow
+    @ViewBuilder
+    private func contentLetterBarView(content: some View) -> some View {
+        if letterPickerEnabled && viewModel.filterViewModel != nil {
```

Comment:
```suggestion
        if letterPickerEnabled, let filterViewModel = viewModel.filterViewModel {
```

and use `filterViewModel` instance instead of force unwrapping below.